.travis.yml

```diff
@@ -10,5 +10,5 @@ before_script:
   - npm install -g grunt-cli istanbul codeclimate-test-reporter
 script:
   - grunt lint
-  - istanbul cover ./node_modules/.bin/_mocha ./spec/helper.js spec/lib/**/*.js --report lcovonly -- -R spec
-  - CODECLIMATE_REPO_TOKEN=88ec807c5d22e853e1a21b9289233f525e793f09e5e81322899ee519587d3689 codeclimate < coverage/lcov.info
+  - istanbul cover ./node_modules/.bin/_mocha spec/helper.js spec/lib/*.js spec/lib/**/*.js --report lcovonly -- -R spec
+  - CODECLIMATE_REPO_TOKEN=0781eab76706eb191ddfadc0a86dc3ad75fa34a9996d6c6e55a6e5fc454ba520 codeclimate < coverage/lcov.info
```

Comment:
I thought that one was the good one? 

https://codeclimate.com/repos/5537d1b369568050720001bc/coverage_setup

Can you elaborate, is the one you pasted the correct one? because I see a different one in the link above.